[Doc][Fluentd]: Update the Fluentd log collection doc (#5798)
DataDog/integrations-core: fluentd/README.md

  dd_tags '<KEY1:VALUE1>,<KEY2:VALUE2>'
  dd_sourcecategory '<SOURCE_CATEGORY>'

  <buffer>
          @type memory
          flush_thread_count 4
          flush_interval 3s
          chunk_limit_size 5m
          chunk_limit_records 500
  </buffer>
</match>
```

By default, the plugin is configured to send logs through HTTPS (port 443) using gzip compression.

Additional parameters can be used to change the endpoint used in order to go through a proxy:

> - `host`: Proxy endpoint when logs are not directly forwarded to Datadog (default value is `http-intake.logs.datadoghq.com`)

Reviewer: ```suggestion
- `host`: The proxy endpoint for logs not directly forwarded to Datadog (default value: `http-intake.logs.datadoghq.com`).
```

> - `port`: Proxy port when logs are not directly forwarded to Datadog (default value is `80`)

Reviewer: ```suggestion
- `port`: The proxy port for logs not directly forwarded to Datadog (default value: `80`).
```

> - `ssl_port`: Port used when logs are forwarded in a secure TCP/SSL connection to Datadog (default is `443`)

Reviewer: ```suggestion
- `ssl_port`: The port used for logs forwarded with a secure TCP/SSL connection to Datadog (default value: `443`).
```

> - `use_ssl`: If `true`, the Agent initializes a secure TCP/SSL connection to Datadog. (default value is `true`)

Reviewer: ```suggestion
- `use_ssl`: Instructs the Agent to initialize a secure TCP/SSL connection to Datadog (default value: `true`).
```

> - `no_ssl_validation`: If you need to disable SSL hostname validation (default value is `false`)

Reviewer: ```suggestion
- `no_ssl_validation`: Disables SSL hostname validation (default value: `false`).
```
